packages/ds-medicare-gov/src/components/Navbar/Navbar.stories.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import MedicaregovLogo from '../MedicaregovLogo';
+import NavigationMenu from '../NavigationMenu';
+import Navbar from './Navbar';
+
+export default {
+  title: 'Components/Navbar',
+  component: Navbar,
+  argTypes: {
+    className: {
+      control: { type: 'text' },
+      type: { name: 'string', required: false },
+    },
+  },
+  args: {
+    children: (
+      <>
+        <div className="m-c-navbar__section">
+          <a href="" className="mct-c-headerLogo">
+            <MedicaregovLogo className="mct-c-headerLogo__image" fill="#12890E" />
+          </a>
+          <div className="m-c-navbar__productName">Medicare Coverage Tools</div>
+        </div>
+        <div className="m-c-navbar__section m-c-navbar__section--right">
+          <NavigationMenu>
+            <li className="m-c-navigationMenu__item">
+              <a href="#somewhere">A link to somewhere</a>
+            </li>
+          </NavigationMenu>
+        </div>
+      </>
+    ),
+    className: '',
+    inverse: true,
+  },
+};
+
+const Template = ({ data, ...args }) => <Navbar {...args} />;
+
+export const BasicNavbar = Template.bind({});
```

Comment:
Currently we use 'default' as the naming convention for this context, can change Basic to Default for these 3?

Everything else looks good!

Reply:
@forrestbaer, I've updated the stories with https://github.com/CMSgov/design-system/pull/1679/commits/86603fe91da10cf7c29b556ff8f565ebc134369b